pepper-apis/dss-core/src/main/java/org/broadinstitute/ddp/service/PdfGenerationService.java

```diff
@@ -335,6 +338,23 @@ Map<Long, ActivityResponse> loadActivityInstanceData(Handle handle, PdfConfigura
         return assignments;
     }
 
+    private void handlePancanReleaseV2(PdfConfiguration config, Map<String, Set<String>> acceptedActivityVersions) {
+        if (config.getStudyGuid().equalsIgnoreCase("cmi-pancan")) {
+            if (config.getConfigName().equalsIgnoreCase("countmein-release")
+                    && acceptedActivityVersions.get("RELEASE").contains("v2")) {
+                acceptedActivityVersions.get("CONSENT").add("v1");
+            } else if (config.getConfigName().equalsIgnoreCase("countmein-release-parental")
+                    && acceptedActivityVersions.get("RELEASE_MINOR").contains("v2")
```

Comment:
Can you pull these strings out to constants?  Some of them might be in use elsewhere.